src/components/Hoverable/index.js

```diff
@@ -1,147 +1,101 @@
-     */
-    setIsHovered(isHovered) {
-        if (this.props.disabled) {
-            return;
-        }
-
-        if (isHovered !== this.state.isHovered) {
-            this.setState({isHovered}, isHovered ? this.props.onHoverIn : this.props.onHoverOut);
-        }
-    }
-
-    /**
-     * Checks the hover state of a component and updates it based on the event target.
-     * This is necessary to handle cases where the hover state might get stuck due to an unreliable mouseleave trigger,
-     * such as when an element is removed before the mouseleave event is triggered.
-     * @param {Event} e - The hover event object.
-     */
-    checkHover(e) {
-        if (!this.wrapperView || !this.state.isHovered) {
-            return;
-        }
-
-        if (this.wrapperView.contains(e.target)) {
-            return;
-        }
-
-        this.setIsHovered(false);
-    }
-
-    handleVisibilityChange() {
-        if (document.visibilityState !== 'hidden') {
-            return;
-        }
-
-        this.setIsHovered(false);
-    }
-
-    render() {
-        let child = this.props.children;
-        if (_.isArray(this.props.children) && this.props.children.length === 1) {
-            child = this.props.children[0];
-        }
-
-        if (_.isFunction(child)) {
-            child = child(this.state.isHovered);
-        }
-
-        if (!DeviceCapabilities.hasHoverSupport()) {
-            return child;
-        }
-
-        return React.cloneElement(React.Children.only(child), {
-            ref: (el) => {
-                this.wrapperView = el;
-
-                // Call the original ref, if any
-                const {ref} = child;
-                if (_.isFunction(ref)) {
-                    ref(el);
-                    return;
-                }
-
-                if (_.isObject(ref)) {
-                    ref.current = el;
-                }
-            },
-            onMouseEnter: (el) => {
-                this.setIsHovered(true);
-
-                if (_.isFunction(child.props.onMouseEnter)) {
-                    child.props.onMouseEnter(el);
-                }
-            },
-            onMouseLeave: (el) => {
-                this.setIsHovered(false);
-
-                if (_.isFunction(child.props.onMouseLeave)) {
-                    child.props.onMouseLeave(el);
-                }
-            },
-            onBlur: (el) => {
-                // Check if the blur event occurred due to clicking outside the element
-                // and the wrapperView contains the element that caused the blur and reset isHovered
-                if (!this.wrapperView.contains(el.target) && !this.wrapperView.contains(el.relatedTarget)) {
-                    this.setIsHovered(false);
-                }
-
-                if (_.isFunction(child.props.onBlur)) {
-                    child.props.onBlur(el);
-                }
-            },
-        });
-    }
+
+function Hoverable({disabled, onHoverIn, onHoverOut, children}) {
```

Comment:
Not seeing that you are exporting the `Hoverable` component

Reply:
Done ✅